pkg/installer/generateconfig.go

```diff
@@ -287,3 +290,14 @@ func (m *manager) generateInstallConfig(ctx context.Context) (*installconfig.Ins
 
 	return installConfig, image, err
 }
+
+func determineVMNetworkingType(vmSku *mgmtcompute.ResourceSku) string {
+	var vmNetworkingType azuretypes.VMNetworkingCapability
+
+	if computeskus.HasCapability(vmSku, azuretypes.AcceleratedNetworkingEnabled) {
+		vmNetworkingType = azuretypes.VMnetworkingTypeAccelerated
```

Comment:
Does this actually work with lower-case `n` on `networking`?

Reply:
This is a typo in openshift/installer.